metrics/mse/requirements.txt

```diff
@@ -1,2 +1,2 @@
 git+https://github.com/huggingface/evaluate@{COMMIT_PLACEHOLDER}
-scikit-learn
\ No newline at end of file
+scikit-learn>=1.5.2
```

Comment:
I am not seeing `root_mean_squared_error` in version earlier than this, hence the new min requirement version